Schlange.py

```diff
@@ -84,42 +84,42 @@ class KeywordTranslator:
     'KeyboardInterrupt': 'Tastaturunterbrechung',
     }
     errors_with_placeholders = {
-    'unsupported operand type': 'nicht unterstützter Operandentyp',
-    'division by zero': 'Division durch Null',
-    'name \'{}\' is not defined': 'Name \'{}\' ist nicht definiert',
-    'list index out of range': 'Listenindex außerhalb des gültigen Bereichs',
-    'tuple index out of range': 'Tupelindex außerhalb des gültigen Bereichs',
-    'invalid syntax': 'Ungültige Syntax',
-    'indentation error': 'Einrückungsfehler',
-    'file not found': 'Datei nicht gefunden',
-    'attribute \'{}\' not found': 'Attribut \'{}\' nicht gefunden',
-    'module \'{}\' not found': 'Modul \'{}\' nicht gefunden',
-    'division or modulo by zero': 'Division oder Modulo durch Null',
-    'key error': 'Schlüsselfehler',
-    'invalid literal for int() with base {}: \'{}\'': 'Ungültiges Literal für int() mit Basis {}: \'{}\'',
-    'unexpected indent': 'Unerwartete Einrückung',
-    'not a valid identifier': 'Kein gültiger Bezeichner',
-    'invalid syntax, unexpected {}: \'{}\'': 'Ungültige Syntax, unerwartetes {}: \'{}\'',
-    'unsupported operand type(s) for {}: \'{}\' and \'{}\'': 'Nicht unterstützte Operandentypen für {}: \'{}\' und \'{}\'',
+    'unsupported operand type': 'nicht unterstützter operandentyp',
+    'division by zero': 'division durch null',
+    'name \'{}\' is not defined': 'name \'{}\' ist nicht definiert',

```

Comment:
All of the english errors starts with a lower-case letters, so the german ones should as well